packages/graphql-kimera-docs/src/pages/index/Feature.js

```diff
@@ -49,7 +49,7 @@ function Feature({ imageUrl, title, description, code, metastring = '' }) {
       <FlexColumn className="col col--4">
         {imageUrl && <FeatureImg src={useBaseUrl(imageUrl)} alt={title} />}
         <h3>{title}</h3>
-        <p>{description}</p>
+        <>{description}</>
```

Comment:
Kinda weird that this doesn't trigger a warning and the above shorthand fragment did.